### PR TITLE
Add some details to the changelog

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## [Unreleased]
 ### Added
 - Mail filtering configured via the JMAP protocol
+- JPA support for Sieve script storage
```
Comment (rouazana) [inline CHANGELOG.md]: you should add "(thanks to...)", that's always nice to thank contributors!

Comment (chibenwa) [reply CHANGELOG.md]: Good habit I agree

**Comment by trantienduchn** [comment]
```
[c20203004a7603cb083f9caad05b68d66f9dda01] [ERROR] Failures: 
[c20203004a7603cb083f9caad05b68d66f9dda01] [ERROR]   DistantMailboxPathRegisterTest.unregisterMixedWithRegisterShouldWorkInAConcurrentEnvironment:328 
[c20203004a7603cb083f9caad05b68d66f9dda01] Expecting:
[c20203004a7603cb083f9caad05b68d66f9dda01]  <{namespace:user:name=200L}>
[c20203004a7603cb083f9caad05b68d66f9dda01] to contain:
[c20203004a7603cb083f9caad05b68d66f9dda01]  <[MapEntry[key=namespace:user:name, value=100L]]>
[c20203004a7603cb083f9caad05b68d66f9dda01] but could not find:
[c20203004a7603cb083f9caad05b68d66f9dda01]  <[MapEntry[key=namespace:user:name, value=100L]]>
[c20203004a7603cb083f9caad05b68d66f9dda01] 
[c20203004a7603cb083f9caad05b68d66f9dda01] [INFO] 
[c20203004a7603cb083f9caad05b68d66f9dda01] [ERROR] Tests run: 476, Failures: 1, Errors: 0, Skipped: 0
```
test this please

**Comment by chibenwa** [comment]
```
[c20203004a7603cb083f9caad05b68d66f9dda01] [ERROR]   SpamAssassinContract.spamShouldBeDeliveredInSpamMailboxWhenSameMessageHasAlreadyBeenMovedToSpam:136 » ConditionTimeout
```

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**